.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Test
+on:
+
+  pull_request:
+  push:
+    branches: [ "REL42.2.5_crate" ]
+
+  # Allow job to be triggered manually.
+  workflow_dispatch:
+
+# Cancel in-progress jobs when pushing to the same branch.
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  test:
+    name: "Java ${{ matrix.java-version }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "ubuntu-latest" ]
+        java-version: [ "8", "11", "17", "19", "20" ]
```

Comment:
why did you chose those? I think 8, 11, 17 as LTS and current one (20) are enough.

Reply:
Changed it. Thanks!